hicommonwealth/commonwealth
# Improving sidebar navigation on a scoped page

## Link to Issue
Closes: #7988

## Description of Changes
- Update topic naming conventions to show relevant icons for gated topics, active contests, or both.
- Changes order of toggle.

## "How We Fixed It"


## Test Plan
- Go to community page.
- Verify that topic are sorted in alphabetical order.
- Verify the topic icon conventions.
  - When a topic is created the default is the “#,”
  - When A topic is gated, it has a lock icon.
  - If the topic is in an active contest, it will have a trophy icon.
  - If a topic is gated AND active contest, the trophy icon is used.

## Deployment Plan

1. 

## Other Considerations

- https://github.com/hicommonwealth/commonwealth/issues/7988#issuecomment-2575897844
